coding_style/README.adoc

```diff
@@ -23,10 +23,15 @@ Rationale:: even when Ansible currently allows names that are not valid identifi
 Making all names valid identifiers will avoid encountering problems in the future. Dictionary keys that are not valid identifiers are also less intuitive to use in Jinja2 (a dot in a dictionary key would be particularly confusing).
 ====
 
+* Use human-readable variable names
```

Comment:
I think that's a duplicate of the next bullet, either skip it or somehow merge it.